docs/configuration/configuration.md

```diff
@@ -260,7 +260,7 @@ job_name: <job_name>
 # OpenMetricsText1.0.0, PrometheusText0.0.4, PrometheusText1.0.0.
 [ scrape_protocols: [<string>, ...] | default = <global_config.scrape_protocols> ]
 
-# Fallback protocol to use if a scrape returns blank, unparseable, or otherwise
+# Fallback protocol to use if a scrape returns blank, unparsable, or otherwise
```

Comment:
Note that "unparseable" is also a fairly common spelling (but less common than "unparsable").